config/polyfills.js

```diff
@@ -8,6 +8,9 @@ if (typeof Promise === 'undefined') {
   window.Promise = require('promise/lib/es6-extensions.js');
 }
 
+// for webpack 5
+require('url')
```

Comment:
Works for me with just the "url" package installed. So maybe this part is redundant?

Reply:
Same for me